src/pyiron_lammps/output.py

```
@@ -183,9 +131,12 @@ def _parse_dump(
     remap_indices_funct: callable = remap_indices_ase,
 ) -> Dict:
     if os.path.isfile(dump_h5_full_file_name):
-        return _collect_dump_from_h5md(
+        if not _check_ortho_prism(prism=prism):
+            raise RuntimeError(
+                "The Lammps output will not be mapped back to pyiron correctly."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)